modules/statics/src/coins.ts

```diff
@@ -390,6 +390,7 @@ export const coins = CoinMap.fromCoins([
   erc20('fwb', 'Friends With Benefits Pro', 18, '0x35bd01fc9d6d5d81ca9e055db88dc49aa2c699a8', UnderlyingAsset.FWB),
   erc20('fxrt', 'FXRT', 3, '0x506742a24c54b77c5af4065b2626ab96c641f90e', UnderlyingAsset.FXRT),
   erc20('fxs', 'Frax Share', 18, '0x3432b6a60d23ca0dfca7761b7ab56459d9c964d0', UnderlyingAsset.FXS),
+  erc20('gal', 'Project Galaxy', 18, '0x5faa989af96af85384b8a938c2ede4a7378d9875', UnderlyingAsset.GAL),
```

Comment:
Isn't GAL the ticker for Gallant?

Reply:
No. As you can check in etherscan is Project Galaxy. https://etherscan.io/token/0x5faa989af96af85384b8a938c2ede4a7378d9875#readContract

Reply:
If we have Gallant as well (in the future), this would cause a conflict in the code base.

Reply:
This is also an issue since erc20 tokens don't lock in unique names themselves either with our current coins map implementation.